common-arcana.lic

```diff
@@ -109,22 +109,31 @@ module DRCA
     end
   end
 
-  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false)
+  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false, runestone_name = nil, runestone_tm = false)
```

Comment:
Is there any other way to do this? 
My fear is that every new option requires optional arguments.

Reply:
I tried other ways and they did not work. Let me see if I can play with this again this weekend.

Reply:
Don't put a lot of weight on it. You're just a victim of my campaign for less optional arguments